modules/ROOT/pages/understanding-enterprise-integration-patterns-using-mule.adoc

```diff
@@ -0,0 +1,92 @@
+
+By implementing these patterns, Mule reduces the effort required when building integrations. You can use these well-known patters to design your solution and then simply configure and use these same patters in Mule.
+
+
+== Mapping Enterprise Integration Patterns into Mule Objects
+
+There is a list of patterns that can be mapped directly with Mule objects. Review the following list of Enterprise Integration Patterns mapped to Mule objects:
+
+=== Integration Styles
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|File Transfer |xref:connectors::file/file-connector.adoc[File Connector]
+|Shared Database |xref:connectors::db/db-connector-index.adoc[DataBase Connector]
+|Remote Procedure Invocation | Mule APIs are meant to work like this (or even doing requests to external APIs).
+|Messaging |Mule is all about Messaging.
+|===
+
+
+=== Messaging Systems
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|Message Channel |Mule provides a message channel that connects the message processors in a flow.
+|Pipes and Filters |A flow implements a pipe and filter architecture.
+|Message Router |xref:about-components.adoc#flow-control-routers[Message Routers].
+|Message Translator |xref:transform-component-about.adoc[Message Transformer].
+|Message Endpoint | xref:mule-sdk::sources.adoc[Message Sources] and xref:mule-sdk::operations.adoc[Operations].
+|===
+
+=== Messaging Channels
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|Point-to-Point Channel |The default channel within a flow.
+|Message Bus |Mule is a message bus.
+|Guaranteed Delivery | Using xref:reliability-patterns.adoc[Reliability Patterns].
+|===
+
+=== Message Construction
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|Event Message | Mule works transmitting events from different Application or Processors.
+|Request Reply | Connectors that allow request-reply wise operations, or using xref:reliability-patterns.adoc[Reliability Patterns].
+|===
+
+=== Message Routing
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|Content-Based Router |xref:choice-router-concept.adoc[Choice Router].
+|Message Filter |  xref:connectors::validation/validation-connector.adoc[Validation Module].
+|Dynamic Routing| xref:about-components.adoc#flow-control-routers[Message Routers].
+|Scatter Gather | xref:scatter-gather-concept.adoc[Scatter Gather Router].
+|Splitter | xref:for-each-scope-concept.adoc[Foreach Scope] and xref:batch-processing-concept.adoc[Batch].
+|Aggregator | xref:connectors::aggregator/aggregators-module.adoc[Aggregator Module].
+
+|===
+
+=== Message Transformation
+
+[%header,cols="2*"]
+|===
+|Pattern |Mapping to a Mule Object
+|Content Enricher |xref:target-variables.adoc[Target Variables].
+|===
+
+=== Messaging Endpoints
+
+You can declare endpoints to be transactional as needed.
```

Comment:
Why does this final table include introductory text? Should the previous tables include something relevant as well?
GLOBALLY try to provide parallel structure for peer items in a file, to set audience expectations and ease consumption of similar information.